Generated electron-app does not work anymore

Currently generating the hello-world app has multiple problems. 

The first issue is
```
'src/browser/hello-world-contribution.ts(14,10): error TS1239: Unable to resolve signature of parameter decorator when called as an expression.\r\n' +
    "  Argument of type 'undefined' is not assignable to parameter of type 'string'.\r\n" +
```
As far as I can see this is fixed on master already (https://github.com/eclipse-theia/generator-theia-extension/issues/164) but not released yet. 

Testing the generation with a local build gives this error then:
```
electron-app: > electron-app@0.0.0 prepare
electron-app: > theia build --mode development
electron-app: Updated dependencies, please run "install" again
electron-app: npm ERR! Lifecycle script `prepare` failed with error: 
electron-app: npm ERR! Error: command failed 
electron-app: npm ERR!   in workspace: electron-app@0.0.0 
electron-app: npm ERR!   at location: /tmp/my-extension/electron-app 
lerna ERR! execute callback with error
lerna ERR! Error: Command failed: npm run prepare
lerna ERR! Updated dependencies, please run "install" again
lerna ERR! npm ERR! Lifecycle script `prepare` failed with error: 
lerna ERR! npm ERR! Error: command failed 
lerna ERR! npm ERR!   in workspace: electron-app@0.0.0 
lerna ERR! npm ERR!   at location: /tmp/my-extension/electron-app 
lerna ERR! 
lerna ERR! 
lerna ERR! > electron-app@0.0.0 prepare
lerna ERR! > theia build --mode development
lerna ERR! 
lerna ERR! 
lerna ERR!     at /tmp/my-extension/node_modules/execa/index.js:236:11
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
```

--- a/templates/app-electron-package.json
+++ b/templates/app-electron-package.json
@@ -19,7 +19,7 @@
   },
   "devDependencies": {
     "@theia/cli": "<%= params.theiaVersion %>",
-    "electron": "^15.3.5"
+    "electron": "^23.2.4"
   },
   "scripts": {
     "prepare": "theia build --mode development",

